samples/peoplenet_detector/overlay.py

```diff
@@ -0,0 +1,138 @@
+import cv2
+
+from savant.deepstream.drawfunc import NvDsDrawFunc
+from savant.deepstream.meta.frame import NvDsFrameMeta, BBox
+from savant.utils.artist import Position, Artist
+from samples.peoplenet_detector.animation import Animation
+from samples.peoplenet_detector.utils import load_sprite, get_font_scale
+
+
+class Overlay(NvDsDrawFunc):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.font_face = cv2.FONT_HERSHEY_SIMPLEX
+        self.font_scale = get_font_scale(
+            self.counters_height, self.counters_font_thickness, self.font_face
+        )
+
+        self.logo = load_sprite(
```

Comment:
The logo is not a sprite, I think better to add separate function and do not confuse users.

Reply:
Sprite is just a piece of 2d graphics, both logo image and animation frames can be considered sprites.

I think it's fine as it is.